src/scirpy/pl/_logoplots.py

```diff
@@ -0,0 +1,91 @@
+from collections.abc import Sequence
+from typing import Literal
+
+from logomaker import Logo, alignment_to_matrix
+
+from scirpy.get import airr as get_airr
+from scirpy.util import DataHandler
+
+
+@DataHandler.inject_param_docs()
+def logoplot_cdr3_motif(
+    adata: DataHandler.TYPE,
+    chains: Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"] | Sequence[Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"]] = "VDJ_1",
+    airr_mod="airr",
+    airr_key="airr",
+    chain_idx_key="chain_indices",
+    cdr3_col: str = "junction_aa",
+    to_type: Sequence[Literal["information", "counts", "probability", "weight"]] = "information",
+    pseudocount: float = 0,
+    background=None,
```

Comment:
can we have a type hint for that? probably it's a np.ndarray?

Reply:
I think you're referring to background here, right? The following screenshot is from logomaker api:
![image](https://github.com/user-attachments/assets/a519e686-b094-436d-8388-1175224b9321)
It seems that it accepts both np.ndarray or pd.dataframe...how can I typeset this accordingly? 💭

Reply:
```suggestion
    background: np.ndarray | pd.DataFrame =None,
```